ios/RN/RNCamera.m

```diff
@@ -92,7 +88,7 @@ - (id)initWithBridge:(RCTBridge *)bridge
         self.cameraId = nil;
         self.isFocusedOnPoint = NO;
         self.isExposedOnPoint = NO;
-        self.invertImageData = false;
+        self.invertImageData = true;
```

Comment:
why this change?

Reply:
If the scanning mode is set to `ALTERNATE`, this flag gets inverted. This change will make sure that the first time the image is scanned, this flag is set to false and thus we try to scan the image first as a normal image.